### ISSUE TITLE
Empty tiles with flat world

### ISSUE THREAD

**Issue by Doelia** [overviewer/Minecraft-Overviewer#2011]
I created a world with Minecraft 1.8.1 vanilla : "Solo / Create new world / More world Options... / World Type : Superflat", defaults settings.

Overviewer generate empty tiles.

<img width="867" alt="Screenshot 2021-12-15 at 19 23 54" src="https://user-images.githubusercontent.com/4354254/146243643-09374513-8024-4535-b6ce-1e8116e8cb3d.png">

It works with a normal world.
I try with Windows binaries and compiling from source on Debian.

No error or warning messages :
```
overviewer.py "world" ./render
2021-12-15 18:25:23  Welcome to Minecraft Overviewer version 0.17.40 (f6ee2cf)!
2021-12-15 18:25:23  Generating textures...
2021-12-15 18:25:42  Preprocessing...
2021-12-15 18:26:01  Rendering 581 total tiles.
 99% [======================================================================================================= ] 577 11.20t/s eta 00h 00m 00s
2021-12-15 18:26:53  Rendering complete!
2021-12-15 18:26:54  Your render has been written to '/render', open index.html to view it.
```

**Comment by spacegriffin** [comment]
Do superflat worlds in 1.18 generate below y=0? If so, that's the problem. Overviewer doesn't currently render blocks below y=0 or above y=256

**Comment by greatmastermario** [comment]
Yes, they generate from Y=-64 as of 1.18. This is fixed in my PR (see screenshots) https://github.com/overviewer/Minecraft-Overviewer/pull/2012

**Comment by lostallmymoney** [comment]
Normal world and I get the same result on 1.18.1
